src/node/full_node.hpp

```diff
@@ -112,14 +112,14 @@ class FullNode : public std::enable_shared_from_this<FullNode> {
   }
 
   template <typename T, typename... ConstructorParams>
-  auto &emplace(std::shared_ptr<T> &ptr, ConstructorParams &&...ctor_params) {
+  auto &emplace(std::shared_ptr<T> &ptr, ConstructorParams &&... ctor_params) {
     ptr = std::make_shared<T>(std::forward<ConstructorParams>(ctor_params)...);
     register_s_ptr(ptr);
     return ptr;
   }
 
   template <typename T, typename... ConstructorParams>
-  static auto &emplace(std::unique_ptr<T> &ptr, ConstructorParams &&...ctor_params) {
+  static auto &emplace(std::unique_ptr<T> &ptr, ConstructorParams &&... ctor_params) {
```

Comment:
I remember the format is for clang-10. We have changed to clang-12 already. Please update clang to version 12 do clang-format

Reply:
I went ahead and did it now.